### ISSUE TITLE
Renderings with Motion Blur?

### ISSUE THREAD

**Issue by vye16** [y-zheng18/point_odyssey#11]
Hey, thank you for the great work! Would it be possible to render videos with motion blur (from both the camera and/or moving objects) similar to how the MOVi-f dataset is generated in kubric (https://github.com/google-research/kubric/blob/main/challenges/movi/README.md#movi-f)? Thanks in advance.

**Comment by y-zheng18** [comment]
Yes, it's possible. You can add nodes to enable motion blur effects in blender, like this: https://github.com/google-research/kubric/blob/main/kubric/renderer/blender_utils.py#L114.
